app/infrastructure/exceptions/base_exception.py

```
@@ -0,0 +1,17 @@
+from typing import Optional
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)